docs/staticAnalysis.md

```diff
@@ -123,6 +123,9 @@ To set it up, [install node.js](https://nodejs.org/en/download/) if necessary (v
 # Alternatively, if you want to install the ESLint module globally, use the install command manually
 # Remember to use the correct tool version
 npm install -g eslint@{version}
+
+# Install eslint-plugin-json for eslint
```

Comment:
Since the `./gradlew installEslint` already include `eslint-plugin-json`, this should be brought together with alternative approach. Something like
```
  # Alternatively, if you want to install the ESLint module globally, use the install command manually
  # Remember to use the correct tool and plugin version
  npm install -g eslint@{version} eslint-plugin-json@{version}
```